packages/ndla-ui/src/ResourcesWrapper/ResourcesTopicTitle.tsx

```diff
@@ -1,17 +1,17 @@
-import React, { Fragment } from 'react';
-import PropTypes from 'prop-types';
+import React, { Fragment, ReactNode } from 'react';
 import { css } from '@emotion/core';
 import styled from '@emotion/styled';
 import { spacing } from '@ndla/core';
 import { HelpCircleDual } from '@ndla/icons/common';
+//@ts-ignore
 import Modal, { ModalBody, ModalHeader, ModalCloseButton } from '@ndla/modal';
 import Tooltip from '@ndla/tooltip';
 import { Switch } from '@ndla/switch';
 import { useTranslation } from 'react-i18next';
 
 import { classes } from './ResourcesWrapper';
 
-const HelpIcon = ({ invertedStyle }) => (
+const HelpIcon = ({ invertedStyle }: { invertedStyle: boolean }) => (
```

Comment:
Ikkje en egen props-type?

Reply:
Så ikke for meg at den skulle få noe særlig flere props, men kan legge inn.

Reply:
Er meir konsekvent slik. Vi har jo propsobjekt med en property andre plasser